### PR TITLE
Bump SpeziAccount

### PR DESCRIPTION
**Pull request by Supereg**
# Bump SpeziAccount

## :recycle: Current situation & Problem
This PR updates the SpeziAccount and SpeziFirebase dependencies fixing several small issue.

## :gear: Release Notes 
* Upgrade SpeziAccount and SpeziFirebase

## :books: Documentation
--

## :white_check_mark: Testing
--

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: TemplateApplication/Supporting Files/Info.plist
Comment (PSchmiedmayer) [inline TemplateApplication/Supporting Files/Info.plist:1]: Thank you @Supereg; I would suggest that we set the Firebase Method Swizzling disable flag here until we remove Firebase from the template application in the future?

File: TemplateApplication.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved
Comment (PSchmiedmayer) [inline TemplateApplication.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: Wondering if we should keep the Package.resolved file committed to ensure more reproducible builds even as dependencies update?

**Review by PSchmiedmayer — APPROVED** [review]
Thank you for pulling the updates into the Template Application. Happy to merge this once the conversation elements are resolved & the tests are passing 👍

File: TemplateApplication/Supporting Files/Info.plist
Comment (Supereg) [reply TemplateApplication/Supporting Files/Info.plist:1]: Added the key 👍 There is nothing we need to add otherwise to keep Firebase working properly?

**Pull request merged**